src/util/heap/layout/vm_layout_constants.rs

```diff
@@ -87,7 +87,7 @@ pub const AVAILABLE_END: Address = HEAP_END;
 pub const AVAILABLE_BYTES: usize = AVAILABLE_END.get_extent(AVAILABLE_START);
 
 /** Granularity at which we map and unmap virtual address space in the heap */
-pub const LOG_MMAP_CHUNK_BYTES: usize = 20;
+pub const LOG_MMAP_CHUNK_BYTES: usize = LOG_BYTES_IN_CHUNK;
```

Comment:
What is the reason for changing this?

Reply:
`LOG_MMAP_CHUNK_BYTES = 20` means you have to do 4 mmap calls to map a 4mb chunk. And Mmappers (e.g. FragmentedMapper) have a table to track the mapping state for each MMAP_CHUNK. So it can map the 1mb MMAP_CHUNK only if it is not mapped before.

By setting the size of a MMAP_CHUNK equals to a 4MB CHUNK, we can directly use the mapping state table to check whether we need to map metadata memory as well.

Reply:
@steveblackburn Is there any reason that a mmap chunk is 1MB and a chunk is 4MB? Is it okay if we change the mmap chunk size to 4MB as well?

Reply:
I don't know the history behind that particular decision.

However, the danger is that this will cause us to significantly overstate our physical memory use.